source/_integrations/fyta.markdown

```diff
@@ -18,7 +18,7 @@ The **FYTA** {% term integration %} uses the open API of [FYTA](https://www.fyta
 
 For the authentication on the FYTA server, you need your login-credentials (email and password).
 
-The integration provides a device for all plants with a FYTA Beam sensor.
+The integration provides a device for all plants with a [FYTA Beam](https://fyta.de/collections/all/products/single-beam) sensor. In order to be able to access your plant data over the API, you need a [FYTA hub](https://fyta.de/collections/all/products/single-hub), that uploads the data from the Beam sensor to the FYTA server. No direct connection to the FYTA Beam is supported.
```

Comment:
The only comment here is that I am unsure if you need a FYTA hub perse. I believe the service can also work with the mobile app.

![CleanShot 2024-04-05 at 17 52 13@2x](https://github.com/home-assistant/home-assistant.io/assets/429659/a9d9415c-db46-432e-8145-d99f08fe0f8d)

Reply:
I'm not sure if the route via mobile app is possible and therefore asked the question to FYTA team. Anyway, I don't think that the option without hub is viable for HA integration, as only the current values and not the history is downloaded. So you would need to regularly connect with your app to the sensors, which does not make sense for me.

Reply:
You were right: https://discord.com/channels/1052218149489094676/1052218150193725452/1225847852039671959
I amended the wording accordingly.

Reply:
Great looks good. 

Don't seem to have access to the discord channel btw.

Ps: It would be possible to connect to Bluetooth to the sensors. That is what brought up the question for me. Save money perhaps by connecting directly to the sensors with HA omitting the hub. 
The website of Fyta states that, well, to have regular updates from the sensor your phone would need to always be near due to the Bluetooth connection required. The thinking is that the HA is always closeby, so perhaps it would be a possibility. This would still leave a gap in terms of how the data arrives in the Fyta DB 🙃